packages/twenty-server/src/engine/workspace-manager/workspace-sync-metadata/constants/standard-field-ids.ts

```diff
@@ -94,7 +94,7 @@ export const calendarEventStandardFieldIds = {
   location: '20202020-641a-4ffe-960d-c3c186d95b17',
   iCalUID: '20202020-f24b-45f4-b6a3-d2f9fcb98714',
   conferenceSolution: '20202020-1c3f-4b5a-b526-5411a82179eb',
-  conferenceUri: '20202020-0fc5-490a-871a-2df8a45ab46c',
+  conferenceLink: '20202020-0fc5-490a-871a-2df8a45ab46c',
```

Comment:
Let's change the uuid here since you are changing the field type we want to create an entirely new field to avoid issues with migration/sync

Reply:
Done